test/e2e/e2ethanos/services.go

```diff
@@ -166,6 +166,36 @@ func NewPrometheusWithSidecarCustomImage(e e2e.Environment, name, promConfig, we
 	return prom, sidecar
 }
 
+func NewPrometheusWithJaegerTracingSidecarCustomImage(e e2e.Environment, name, promConfig, webConfig,
```

Comment:
I have changed this to the original but the E2E tests still fail and I don't see a panic/crash in the logs, so I _think_ it could be something unrelated to my changes?